FindAietools.cmake

```diff
@@ -0,0 +1,216 @@
+###############################################################################
+#  Copyright (c) 2024, AMD, Inc.
+#  All rights recd cd cd served.
+# 
+#  Redistribution and use in source and binary forms, with or without 
+#  modification, are permitted provided that the following conditions are met:
+#
+#  1.  Redistributions of source code must retain the above copyright notice, 
+#     this list of conditions and the following disclaimer.
+#
+#  2.  Redistributions in binary form must reproduce the above copyright 
+#      notice, this list of conditions and the following disclaimer in the 
+#      documentation and/or other materials provided with the distribution.
+#
+#  3.  Neither the name of the copyright holder nor the names of its 
+#      contributors may be used to endorse or promote products derived from 
+#      this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, 
+#  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR 
+#  PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR 
+#  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, 
+#  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, 
+#  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+#  OR BUSINESS INTERRUPTION). HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, 
+#  WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR 
+#  OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF 
+#  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+###############################################################################
+
+###########################################################
+#
+## 1: Setup:
+# The following environmental variables are optionally searched for defaults:
+#  none.
+#
+## 2: Variables
+# The following are set after configuration is done: 
+#  
+#  AIETOOLS_DIR - The path to Aietools installation directory for the specified version
+#  AIETOOLS_XCHESSCC - 'xchesscc' with full path
+#  AIETOOLS_XCHESS_MAKE - 'xchessmk' with full path
+#
+## 3: Components
+# The following components are supported:
+###  3.1. AIE
+#  AIETOOLS_AIE_INCLUDE_DIR - AIE full include path
+#  AIETOOLS_AIE_LIBME - 'libme.a' with full path
+#  AIETOOLS_AIE_LIBC - 'libc.a' with full path
+#  AIETOOLS_AIE_LIBM - 'libm.a' with full path
+#  AIETOOLS_AIE_RUNTIME_INCLUDE_DIR - AIE runtime full include path
+#  AIETOOLS_AIE_LIBSOFTFLOAT - 'softfloat.a' with full path
```

Comment:
Should this really just be a refactoring of FindVitis?  So FindVitis should call this?

Reply:
yes

Reply:
FindVitis now calls FindAIETools in a backwards compatible way